src/main/java/detail/datasets/IDataSetContainer.java

```diff
@@ -23,4 +26,15 @@
      * Add a new data set in the container
      */
     boolean addDataSet(IDataSet dataSet);
+
+    /**
+     * Returns  a list of the loaded dataset names
+     */
+    List<String> dataSetNames();
+
+    /**
+     * Clear whatever the data set  may contain
+     */
+    void clear();
```

Comment:
why do we need a clear function here? maybe it's too risky to give the user the option to delete the contents of the data set.

Reply:
This class is meant to fold all the datasets loaded by the app. Once the user clicks on Quit everything should be dumped. This is just a collection of data sets and not an actual data set